src/shogun-core-main/src/main/java/de/terrestris/shoguncore/util/interceptor/WmtsUtil.java

```diff
@@ -0,0 +1,90 @@
+package de.terrestris.shoguncore.util.interceptor;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Locale;
+import java.util.regex.Pattern;
+
+import static org.apache.logging.log4j.LogManager.getLogger;
+
+/**
+ * @author Johannes Weskamm
+ * @author terrestris GmbH & Co. KG
+ */
+public class WmtsUtil {
+
+    /**
+     * The Logger.
+     */
+    private static final Logger LOG = getLogger(WmtsUtil.class);
+
+    /**
+     * Check if given request is a RESTful WMTS call
+     * @param mutableRequest
+     * @return
+     */
+    public static boolean isRestfulWmtsRequest(MutableHttpServletRequest mutableRequest) {
+        String url = mutableRequest.getRequestURL().toString();
+        String service = mutableRequest.getParameterIgnoreCase("service");
+        if (!StringUtils.isEmpty(service) && service.equalsIgnoreCase("wmts")) {
+            // looks like KVP
+            return false;
+        }
```

Comment:
This test seems very fragile, couldn't we at least check for the presence of a pattern like `{TileMatrixSet}/{TileMatrix}/{TileRow}/{TileCol}.png` (like in `isRestfulWmtsGetFeatureinfo()` below)?